front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -47,18 +53,6 @@ const Container = styled.div<{
       transition: all ${slideInOutTimeout}ms ${slideInOutEasing};
     }
   }
-
-  ${(props) => media.tablet`
-    height: 100vh;
-    top: 0;
-    bottom: ${props.theme.mobileMenuHeight}px;
-    z-index: 1005; /* there is no top navbar at this screen size, so okay that it is higher than the z-index of NavBar here */
-
-    &.hasBottomBar {
-      height: ${props.windowHeight}px;
-      bottom: 0;
-    }
-  `}
 `;
```

Comment:
Since this component is only used for tablets or smaller now, this CSS was always used rather than the styles at the top. You can ignore `height` here (dealing with that for the whole component on the right),`top`, `bottom` etc. are not needed. And we're now using the `z-index` (1005) from here for all device widths.